torchquad/tests/gauss_test.py

```diff
@@ -84,22 +74,72 @@ def _run_gaussian_tests(backend, _precision):
 
     # 10D Tests
     N = (60**3) * 3
```

Comment:
Yeah, this was a guess.  I'm not really sure what we should be going for here.

Reply:
N_d points per dim would lead to (N_d - 1) * (degree+1) points in that dimension, right? Then I would propose to maybe go for 3**10 or something like that? For Boole, e.g. , by that logic we picked 5**10 but that is already quite some points :upside_down_face: